lib/icalendar/util/deserialize.ex

```diff
@@ -175,10 +175,10 @@ defmodule ICalendar.Util.Deserialize do
   end
 
   def parse_attr(
-        %Property{key: "ORGANIZER", params: _params, value: organizer},
+        %Property{key: "ORGANIZER", params: params, value: value},
         acc
       ) do
-    %{acc | organizer: organizer}
+    %{acc | organizer: Map.put(params, :original_value, value)}
```

Comment:
What is this original value field? This doesn't seem like something to return to the user.

Reply:
@lpil It is the same as how `ATTENDEE` is currently handled so I just did the same for consistency. See: https://github.com/lpil/icalendar/blob/902df35488ded9310d0f3fc0611b60bab57d8735/lib/icalendar/util/deserialize.ex#L184-L189

Extracted:

```elixir

  def parse_attr(
        %Property{key: "ATTENDEE", params: params, value: value},
        acc
      ) do
    %{acc | attendees: [Map.put(params, :original_value, value)] ++ acc.attendees}
  end
```

What it contains is just the email.

Reply:
If we change how we do it we should probably change ATTENDEES as well.

Reply:
Thank you